checker/tests/resourceleak/CreatesMustCallForTargets.java

```diff
@@ -10,6 +10,7 @@ class CreatesMustCallForTargets {
   @Owning InputStream is1;
 
   @CreatesMustCallFor
+  // :: error: createsmustcallfor.target.unparseable
```

Comment:
This error looks correct, but it worries me that it only started showing up in this PR - "this" was never a valid target for a static method.

Reply:
Before, we would only try to parse the annotation targets in the invocation context, not the method declaration context.  Further, this method is never invoked.  So we didn't get the unparseable error before.  Now we parse the targets in both contexts, so we get an error at this declaration.

Reply:
That explanation makes sense, thanks. I'm glad it doesn't indicate a deeper problem.